### ISSUE TITLE
CWT observation model

### ISSUE THREAD

**Issue by marksorel8** [pbs-assess/stockseasonr#1]
For use with coded wire tag (CWT) data, it would be great if the tag rates of different stocks could be passed as data (e.g., if only 25% of smolts receive CWTs in one hatchery and 10% in another), which would then be used to calculate the expected proportions of tags recovered from the latent proportions of stocks at large.

This would make this a very useful tool for the run reconstruction that we do here in the Columbia River and I suspect for the type of modeling that the CTC does as well. I'd be willing to help with this enhancement.

**Comment by CamFreshwater** [comment]
Sorry for the delay! I'll spend some time mulling over how tagging intensity or similar variables could be incorporated here, but am totally open to suggestions and contributions!

**Comment by catarinawor** [comment]
Yes you need to standardize the recovery data by release size across stocks and brood years. In the CTC we do it before we use the CTC data (although we just do it across BY, as stock composition is not a concern for the cohort analysis). 

The math is: 

AdjRecoveries(BY,S)<- MaxRelease/CWTRelease(BY,S)

Hope this is helpful.

**Comment by marksorel8** [comment]
Thanks! Yeah, in our Columbia River run reconstruction we divide tag recoveries by the tag rate for each stock and then normalize to get proportions of catch by stock. The whole process is described in a paper (see equation 8): 

Jensen, Alexander J., Benjamin Cox, and James T. Peterson. "Evaluating tag-reliant harvest estimators in Chinook salmon mixed-stock fisheries using simulations." Canadian Journal of Fisheries and Aquatic Sciences 99.999 (2022): 1-15.

For our process I don't think (hope) it would be too hard to take the proportions in the process model and translate them to expected proportions of recoveries based on the tag rates, and then those proportions could be used in the multinomial likelihood for the actual tag recovery data.

I'm interested in trying this out when I have time,
